### DISCUSSION TITLE
AOSP 11.0 v307

### DISCUSSION THREAD

**Discussion by phhusson** [phhusson/treble_experimentations#1824] [General]
- May security patch

<hr /><em>This discussion was created from the release <a href='https://github.com/phhusson/treble_experimentations/releases/tag/v307'>AOSP 11.0 v307</a>.</em>

**Comment by Daniel1902-007** [comment]
Magisk?

**Reply by haruki-taka8** [reply]
#1667?

**Comment by babaghanush** [comment]
Any way to easily update it from v306 without installing the rom from scratch ?

**Reply by JonnyTech** [reply]
Simply flash the GSI to your system partition from fastboot or recovery then reboot.

**Reply by babaghanush** [reply]
And it won't wipe my current apps and system settings?

**Reply by JonnyTech** [reply]
No, that data is stored elsewhere (userdata partition) - flashing just the system and preserving data is called a "dirty flash".

**Reply by babaghanush** [reply]
Thank you.  Dirty Flash worked just fine.  I am now on May security patch without losing my apps or settings.

**Comment by ossage** [comment]
Is it a new CAF now, or in call bt bug still exists?

**Comment by savitar00** [comment]
I don't know which one to download just for my proceser it's  Arm v7a

**Comment by brendanus** [comment]
Blackview BV9600E
Just to say many thanks @phhusson for this great GSI! I flashed it (system-roar-arm64-ab-vndklite-gapps.img) on my Blackview BV9600E without problems, after unlocking and the usual preparations. Booted very quickly. No noticeable glitches (other than face unlock, which I don't mind). In fact it is better, less buggy, than the stock rom in several other respects. The SU is very useful. 
The BV9600E (MediaTek Helio P70, MT6771T) shipped with Android 9. The system partition is limited to 2971Mb, so this rom just fits. 
The post-setup involved going to the Google registration page (https://www.google.com/android/uncertified/?pli=1) to submit the Google Services Framework Android ID, as Google now rejects custom roms until the phone is re-registered.

**Comment by nthebach** [comment]
My huawei m5 pro tablet support 9V - 2A fast charging. The gsi has 2 options for huawei devices which change the current to mid-fast (2.75A) and fast (4A). Is there any way to add 9V fast charging support for the device?
